setup.py

```diff
@@ -186,12 +186,45 @@ def check_submodules():
             raise ValueError('Submodule not clean: %s' % line)
 
 
+class concat_license_files():
+    """Merge LICENSE.txt and LICENSE_bundled.txt for sdist creation
+
+    Done this way to keep LICENSE.txt in repo as exact BSD 3-clause (see
+    gh-13447).  This makes GitHub state correctly how NumPy is licensed.
+    """
+    def __init__(self):
+        self.f1 = 'LICENSE.txt'
+        self.f2 = 'LICENSE_bundled.txt'
+
+    def __enter__(self):
+        """Concatenate files and remove LICENSE_bundled.txt"""
+        with open(self.f1, 'r') as f1:
+            self.bsd_text = f1.read()
+
+        with open(self.f1, 'a') as f1:
+            with open(self.f2, 'r') as f2:
+                self.bundled_text = f2.read()
+                f1.write('\n\n')
+                f1.write(self.bundled_text)
+
+        os.remove(self.f2)
```

Comment:
Could just prune this in `MANIFEST.in`?

Reply:
Good suggestion, done. That also made me spot another issue: README.md was not included in the sdist - fixed that as well.